velox/exec/fuzzer/WindowFuzzer.h

```diff
@@ -24,6 +24,9 @@
 
 DECLARE_bool(enable_window_reference_verification);
 
+const int frameTypeRange = 0;
```

Comment:
Instead of defining these new variables, let us reuse the existing enum from `WindowNode`, `WindowNode::WindowType::kRows`.